Home_Work_1/task_2.py

```diff
@@ -0,0 +1,8 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+number = int(input('Введите общее число журавликов: '))
+
+katya = int((number / 3) * 2)
```

Comment:
Не принято использовать в качестве имен переменных транслитерацию русских слов. И тут скобки и функция `int` не нужна.
```suggestion
katya = number // 3 * 2
```